src/System.Text.Json/tests/Serialization/Value.ReadTests.cs

```diff
@@ -365,22 +365,18 @@ private static void AssertFloatingPointBehavior<T>(T netfxExpectedValue, T netco
             }
         }
 
-        // NOTE: LongInputString test is constrained to run on Windows and MacOSX because it causes
```

Comment:
Can we still keep (or maybe add separately as the outerloop) a test like this where the input is closer to int.MaxValue? Basically, anything that would cause `json.Length * MaxExpansionFactorWhileTranscoding` to overflow an int32.

Reply:
This feedback wasn't addressed.

Reply:
This is a separate test case. Due to refactoring that test no longer makes sense with this PR.

Added https://github.com/dotnet/corefx/issues/41687